root/usr/sbin/hotsync-slave

```diff
@@ -26,6 +26,11 @@
 # launch reinstall action
 /etc/e-smith/events/actions/restore-config-reinstall
 
+if [[ $? != 0 ]]; then
+    echo "[ERROR] restore-config-reinstall failed!" 1>&2
+    exit 1
```

Comment:
I'm not sure about that: if restore-config-reinstall fails, it means that there was a problem installing some RPMs, but that doesn't mean that backup-config is bad and maybe it should be on server-manager interface